src/pages/test-report/test-report.ts

```diff
@@ -126,6 +131,19 @@ export class TestReportPage extends BasePageComponent {
 
   onEndTestClick = (): void => {
     console.log('Clicked End Test');
+    const zoomClass = `${this.app.getTextZoomClass()}`;
+    // MES 2196 to return valid state
```

Comment:
These comments could be prefixed with `TODO:` so that we can find them later on

Reply:
Done